spec/lib/manageiq/providers/workflows/builtin_methods_spec.rb

```diff
@@ -17,14 +17,56 @@
     let(:params)  { {} }
     let(:secrets) { {} }
 
-    it "returns an error if _object_type isn't passed" do
-      runner_context = described_class.provision_execute(params, secrets, Floe::Workflow::Context.new({"Execution" => {"_object_id" => nil}}))
-      expect(runner_context).to include("running" => false, "success" => false, "output" => {"Error" => "States.TaskFailed", "Cause" => "Missing MiqRequestTask type"})
+    it "requires _object_type" do
+      runner_context = described_class.provision_execute(params, secrets, create_floe_context(:execution => {"_object_id" => nil}))
+      expect(runner_context).to include("running" => false, "success" => false, "output" => failed_task_status("Missing MiqRequestTask type"))
     end
 
-    it "returns an error if _object_id isn't passed" do
-      runner_context = described_class.provision_execute(params, secrets, Floe::Workflow::Context.new({"Execution" => {"_object_type" => "MiqProvision"}}))
-      expect(runner_context).to include("running" => false, "success" => false, "output" => {"Error" => "States.TaskFailed", "Cause" => "Missing MiqRequestTask id"})
+    it "requires _object_id" do
+      runner_context = described_class.provision_execute(params, secrets, create_floe_context(:execution => {"_object_type" => "MiqProvision"}))
+      expect(runner_context).to include("running" => false, "success" => false, "output" => failed_task_status("Missing MiqRequestTask id"))
     end
+
+    it "requires a provisioning object" do
+      floe_context = create_floe_context(FactoryBot.create(:service_reconfigure_task, :request_type => "service_reconfigure"))
+      runner_context = described_class.provision_execute(params, secrets, floe_context)
+      expect(runner_context).to include("running" => false, "success" => false, "output" => failed_task_status(/Calling provision_execute on non-provisioning request/))
+    end
+
+    it "requires a provisioning object" do
```

Comment:
These 2 have the same title - not sure if that's a problem or if one clobbers the other.

Reply:
it was a problem. thanks